Automattic/wp-calypso
# Add eCommerce signup flow

#### Changes proposed in this Pull Request

This PR is related to the new ecommerce plan, and as of now it's based on https://github.com/Automattic/wp-calypso/pull/28176. Once #28176 is merged, we'll be able to rebase it onto master to make the changes more clear. Also, we need to have the Blogger PR merged before we can add eCommerce plan to the "business" tab (https://github.com/Automattic/wp-calypso/pull/27125)

The purpose is to add a signup flow for the new plan. Specifically, this PR:
* Renames the `store-nux` signup flow to `ecommerce`
* Removes the `themes` step and defaults to twentyfifteen theme
* De-emphasises `.blog` domain suggestions
* Changes the plan recommendation to ecommerce
* Adds the ecommerce plan to plans list in signup and on /plans

#### TODO:

~~Once Blogger PR is merged, this needs to be rebased.~~ An update will be required for `client/my-sites/plans-features-main/index.jsx` to add the ecommerce plan to the "business plans" tab.

#### Testing instructions

* Create a new site
* Choose "Sell products or collect payments" as a primary goal in signup step 1
* You should be redirected to domains step, does not matter what you do you should never get a recommendation for `.blog` domain, only for other TLDs
* Once you choose a domain, you should be redirected to plan selection with one plan available - ecommerce plan
* Same test steps as in #27125, only with the eCommerce plan (once the TODO section above is addressed)
